pkg/metrics/metrics.go

```diff
@@ -17,8 +17,6 @@ limitations under the License.
 package metrics
 
 import (
-	"math"
-	"strconv"
 	"strings"
 	"sync"
 )
```

Comment:
Is the concept of a `FamilyGenerator` something you would like to use @LiliC, or should we also move that into `/internal`?

Reply:
Yes, I make use the `FamilyGenerator`.